WMIAdventure/frontend/src/js/components/battle/organisms/BattleView/BattleView.js

```diff
@@ -32,19 +39,44 @@ class BattleView extends React.Component {
                            setTranslateY={this.props.setTranslateY}>
                         <MainContainer>
                             <FlexGapContainer setMargin={'10px 0 0 0'}>
-                                <UserStateContainer enemy />
-                                <CompactCardView setWidth={'124px'} setHeight={'192px'} setMargin={'0 0 0 10px'} />
+                                <ColumnGapContainer gap={'0'}>
+                                    <UserStateContainer enemy hp={'99'} shield={'1'} />
+                                    <FlexGapContainer setWidth={'100%'} space>
+                                        <MiniCardView enemy rank={3} visible={false} icon={icon1} />
+                                        <MiniCardView enemy rank={2} visible icon={icon2} />
+                                        <MiniCardView enemy rank={3} visible icon={icon3} />
+                                        <MiniCardView enemy rank={1} visible icon={icon4} />
+                                        <MiniCardView enemy rank={1} visible icon={icon5} />

```

Comment:
Może lepiej pętlą to zrobić? Potem jak będziemy faktycznie mieli karty to myślę że to ułatwi.

Reply:
Ale nie wiem jeszcze, czy nie utrudni manipulację kolejnością